app/models/financialDetails/TransactionType.scala

```diff
@@ -145,6 +153,9 @@ object CreditType {
   private val mfaCredit = Range.inclusive(4004, 4025)
     .filterNot(_ == 4010).filterNot(_ == 4020).map(_.toString)
     .toList
+  private val paymentOnAccountOneReviewAndReconcileCredit = "4912"
```

Comment:
if we can move this up around line 99, to keep transaction codes close to each other